ui/pages/SummaryData/components/SampleMetadata.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { mount, configure } from 'enzyme'
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
+
+import { SampleMetadata } from './SampleMetadata'
+
+configure({ adapter: new Adapter() })
+
+const DATA = [
+  {
+    projectGuid: 'R0003_test',
+    num_saved_variants: 2,
+    solve_state: 'Tier 1',
+    sample_id: 'NA20889',
+    'Gene_Class-1': 'Tier 1 - Candidate',
+    'Gene_Class-2': 'Tier 1 - Candidate',
+    'inheritance_description-1': 'Autosomal recessive (compound heterozygous)',
+    'inheritance_description-2': 'Autosomal recessive (compound heterozygous)',
+    hpo_absent: '',
+    'novel_mendelian_gene-1': 'Y',
+    'novel_mendelian_gene-2': 'Y',
+    'hgvsc-1': 'c.3955G>A',
+    date_data_generation: '2017-02-05',
+    'Zygosity-1': 'Heterozygous',
+    'Zygosity-2': 'Heterozygous',
+    'Ref-1': 'TC',
+    'sv_type-2': 'Deletion',
+    'sv_name-2': 'DEL:chr12:49045487-49045898',
+    'Chrom-2': '12',
+    'Pos-2': '49045487',
+    maternal_id: '',
+    paternal_id: '',
+    maternal_guid: '',
+    paternal_guid: '',
+    'hgvsp-1': 'c.1586-17C>G',
+    project_id: 'Test Reprocessed Project',
+    'Pos-1': '248367227',
+    data_type: 'WES',
+    familyGuid: 'F000012_12',
+    congenital_status: 'Unknown',
+    family_history: 'Yes',
+    hpo_present: 'HP:0011675 (Arrhythmia)|HP:0001509 ()',
+    'Transcript-1': 'ENST00000505820',
+    ancestry: 'Ashkenazi Jewish',
+    phenotype_group: '',
+    sex: 'Female',
+    'Chrom-1': '1',
+    'Alt-1': 'T',
+    'Gene-1': 'OR4G11P',
+    'gene_id-1': 'ENSG00000240361',
+    pmid_id: null,
+    phenotype_description: null,
+    affected_status: 'Affected',
+    analysisStatus: 'Q',
+    filter_flags: '',
+    disorders: null,
+    family_id: '12',
+    displayName: '12',
+    MME: 'Y',
+    subject_id: 'NA20889',
+    individual_guid: 'I000017_na20889',
+    proband_relationship: 'Self',
+    consanguinity: 'None suspected',
+  },
+]
+
+test('SampleMetadataDataTable render and export', () => {
```

Comment:
we do not generally have many or robust client side unit tests so this is atypical for the repo, but given that this file will be a dependency for the computational biologists who support seqr we want to ensure this file is stable (or at least does not change unintentionally) it makes sense to add testing for it

Reply:
I was curious about the enzyme adapter and found some fun developer drama wrt enzyme support for react.. here's a blog post from the adapter maintainer: https://dev.to/wojtekmaj/enzyme-is-dead-now-what-ekl. Is this on your radar? Should we avoid writing enzyme tests in the future and try jest or React Testing Library instead?

Reply:
So thats a really good point, but I think realistically we do not have the bandwidth to invest any substantive dev time in  UI testing given our current workload. So our options in the meantime are basically "continue to write new enzyme tests and then when we do migrate have more tests to migrate" or "do not write any new UI tests". Given those options, I think I prefer the former. We write UI tests so infrequently that the time we will save by having a couple tests already moved over is pretty minimal, and it would be better to just switch over all our tests and deprecate enzyme in one big push.

Reply:
I agree with your conclusion. I like how you list out the concrete options for what we can do Right Now in addition to thinking about something for the future.